tasks/databases.rake

```diff
@@ -367,6 +381,16 @@ def sort_string migration
   "#{migration[:version]}_#{migration[:kind] == :data ? 1 : 0}"
 end
 
+def abort_if_pending_migrations(migrations, message)
+  if migrations.any?
```

Comment:
this rake file has been neglected, so for any new changes, i've been trying not to add more lines to the file. hence, i started putting logic for rake taks into `lib/data_migrate/tasks/data_migrate_tasks.rb`. would you mind adding this in there

Reply:
Sure, done